src/github/client/github-installation-client.ts

```diff
@@ -263,6 +264,18 @@ export class GitHubInstallationClient extends GitHubClient {
 		return response?.data?.data;
 	}
 
+	public async getDeploymentsPageByCreatedAtDesc(owner: string, repoName: string, perPage?: number, cursor?: string | number): Promise<getDeploymentsResponse> {
```

Comment:
Should the cursor always be a string if its a graphql call?
`cursor?: string | number`

Reply:
Make sense. Although I tried to rectify it, but seems involve quite a lots of files. So will keep it to next PR.